Components/Navigation/navigation.css

```diff
@@ -0,0 +1,287 @@
+@import url('/CSS/styleGuide.css');
+@import url('/CSS/sideBar.css');
+@import url('/CSS/home.css');
+@import url('/CSS/reset.css');
+
+/******** Main body **************/
+
+/******** Navigation *******/
+
+.nav--container {
+    margin: 10vh;
+    width: 70vw;
+    margin-top: 15vh;
+}
+
+/* Square image */
+.square__img {
```

Comment:
I dont see any img in nav component, so not sure what you've mentioned in nav's css.